lldb/tools/lldb-dap/JSONUtils.cpp

```diff
@@ -135,6 +135,18 @@ std::vector<std::string> GetStrings(const llvm::json::Object *obj,
   return strs;
 }
 
+static std::string GetDescriptionTrimmed(lldb::SBValue &value) {
+  lldb::SBStream stream;
+  value.GetDescription(stream);
```

Comment:
The method `bool SBValue::GetDescription(SBStream &description)` returns `true` always. Could you make it return false if the description couldn't be generated, and also abort early in this function in such case?
I've also noticed that `GetDescription` dumps `"No value"` in some cases, which would be better not to display via `lldb-dap`.

Reply:
Updated `bool SBValue::GetDescription(SBStream &description)` to return `false` for the `"No value"` case and this helper should return an empty string in that situation fallback back to the `<type> @ <pointer>` format.